cmd/secretd/main.go

```diff
@@ -80,6 +83,26 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
+
+	// Set default gas limit for WASM queries
+	queryGasLimitTemplate := `
```

Comment:
You should edit this:
https://github.com/cosmos/cosmos-sdk/blob/v0.38.3/server/config/toml.go

Instead of putting it here hardcoded

Reply:
It's inside the sdk

Reply:
Yep, don't we use our own fork for a lot of things?

Reply:
Yeah, but I think we only fixed the multisig issue and upgraded v0.38.3 to use Tendermint v0.33.6.
The second link you sent is synced with their master which is somewhere between v0.39 to v0.40.

Essentially this is what we have right now on mainnet: https://github.com/cosmos/cosmos-sdk/compare/v0.38.3...enigmampc:v0.38.3-scrt-0.2.1